app/assets/stylesheets/components/_native-campaign.scss

```diff
@@ -0,0 +1,37 @@
+$light-grey: #bfc1c3;
+$dark-grey: #999;
+
+.native-campaign {
```

Comment:
I think these styles should be prefixed with `app-c-` eg. `app-c-native-campaign`